test/Spec.hs

```diff
@@ -11,6 +11,10 @@ import Data.Maybe (isJust, fromMaybe)
 
 main :: IO ()
 main = hspec $ do
+  describe "parse HEALTHCHECK" $
+    it "parse healthcheck without args" $
+        assertAst "HEALTHCHECK --interval=5m \\nCMD curl -f http://localhost/" [Healthcheck "--interval=5m \\nCMD curl -f http://localhost/"]
```

Comment:
Interesting that the `CMD` is embedded inside the `HEALTHCHECK` instruction. Probably worth a followup PR that parses this `CMD` so it can be linted as well